src/ucp/core/ucp_worker.c

```diff
@@ -2959,13 +2966,28 @@ ucp_worker_do_keepalive_progress(ucp_worker_h worker)
             goto out;
         }
 
+#if UCS_ENABLE_ASSERT
+        ucp_ep_ext_control(ep)->ka_count++;
+        /* Difference between the number of EP keepalive rounds and total number
+         * of KA rounds done on Worker must be <= 2, because keepalive could be
+         * done twice for the same EP in case of number of EPs was decreased */
+        ucs_assertv((ucp_ep_ext_control(ep)->ka_count /
+                             (worker->keepalive.round_count + 1)) <= 2,
```

Comment:
should not it be a difference not a relation?

Reply:
it should be relation, since we could do KA twice for the EP on every round.

Reply:
ok